_includes/markdown/Markup.md

```diff
@@ -285,10 +285,64 @@ Combining automated and manual testing practices allows 10up to maintain a high
 
-At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible. For example, browser support for SVG has not yet reached 100%. When using SVG you should always provide a fallback such as a PNG image for browsers that do not support vector graphics.
+At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible.
 
 ### Polyfills
 When writing markup that does not have wide browser support, using polyfills can help bring that functionality to those older browsers. Providing support for older browsers is incredibly important to the business objectives of our clients. In an effort to prevent code bloat, we only provide polyfills for features that are functionally critical to a site.
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. Here are a few known benefits of SVG:
+
+* __Scalability__ - They look great on retina displays and at any size, i.e. they're resolution independent.
+* __File Size__ - Small file size and compresses well.
+* __Styling__ - Manipulate fill, stroke, and even animate.
+
+### SVG Sprites
+
+It is optimal to group multiple SVG icons that are likely to be used throughout certain templates or even the entire site (i.e. sprite).
+
+Combine them in to an `svg-defs.svg` file, embed the `svg-defs.svg` within the document, and reference them within the template source with `<use>`. The creation of this icon system should be automated. Check out CSS-Tricks.com's write up on this technique: [Icon Systems with SVG Sprites](https://css-tricks.com/svg-sprites-use-better-icon-fonts/)
+
+### SVG embedded in HTML
+
+When your placing an SVG in markup (i.e. inline) be sure to use the following guidelines:
```

Comment:
"your" should be "you're"

Reply:
@dkoo I'm so grateful to have fellow engineers reviewing for my glaring typos. 🙈  Thanks!